### ISSUE TITLE
Entity None implements HVACMode(s)

### ISSUE THREAD

**Issue by stu247** [PeteRager/lennoxs30#304]
**Describe the bug**
I upgraded HA to 2024.2.1 today and saw the following warning in home-assistant.log:
```
2024-02-09 08:43:29.181 WARNING (MainThread) [homeassistant.components.climate] Entity None (<class 'custom_components.lennoxs30.climate.S30Climate'>) implements HVACMode(s): off, cool, heat, heat_cool and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/PeteRager/lennoxs30/issues
```

**To Reproduce**
Restart HA in 2024.2.1.

**Expected behavior**
No warning.

**Screenshots/Logs**
See above.

**Version**
 - Home Assistant: 2024.2.1
 - This Integration: 2024.2.0
 

**Integration Configuration**
- Cloud or Local Connection: local
- Inverter or Sensor Diagnostics Enabled: no

**Additional context**
NA.

**Comment by PeteRager** [comment]
Thanks for the report.

**Comment by PeteRager** [comment]
Fixed in release 2024.2.1
